### ISSUE TITLE
Freezing on desktop & mobile with lots of containers (Due to Live Stats?)

### ISSUE THREAD

**Issue by calvinbui** [amir20/dozzle#1990]
**Describe the bug**

I'm currently running 118 containers on my computer. I've always had freezing issues with Dozzle on desktop and always on mobile. 

These are some screenshots I took earlier on my Windows PC (i7 7700K, 32GB RAM).

![image](https://user-images.githubusercontent.com/3604363/208151383-13418095-e198-4d94-97fb-7282bf5f2938.png)

![image](https://user-images.githubusercontent.com/3604363/208151415-755dcc65-c516-4b2e-a525-daf952c8c204.png)

On my phone, it will either freeze when loading the list of containers or freeze completely when clicking on a container to view its logs.

My suspicion is the CPU and MEM live stats. I've attached a video below. The list items don't get highlighted until the `TOTAL CPU USAGE` counter updates. I think this could be blocking something? 

https://user-images.githubusercontent.com/3604363/208153147-8005f76f-e2af-4970-8ded-1c53c3d21e36.mp4

After disabling 70 containers, the problem was still there, but less so.

https://user-images.githubusercontent.com/3604363/208154500-0fff3669-0a64-46e1-8342-370dfa6dc223.mp4

Another suggestion I have is to provide an option to disable live stats? I'll be happy to give that a try.

**Desktop (please complete the following information):**
 - OS: Windows and Android
 - Docker version 20.10.22
 - Browser & version Chromium on Windows and Bromite & Chrome on Android
 - Version 4.6.0

**If applicable include logs with `--level debug` and browser logs**

Nothing of note in browser and Dozzle logs.

**Comment by amir20** [comment]
That's not good @calvinbui. I take great pride in quality of Dozzle. I had stress tested a while back, but that was before I added the live metrics. 

My first goal is to actually reproduce this.  I only run about 30 containers in my env. Let me create some fake containers and see what happens. If I can't reproduce, I'll need more info.

**Comment by amir20** [comment]
Test 100+ containers on my laptop. The UI is sluggish and moving the mouse around is choppy. 

Clicking each of the containers did load the logs quickly. But that makes sense because I use a different connection to load the logs. 

My initial hypothesis is that it can be the live stats.  I'll try disabling and see if that helps. I'll have to think where the problem can be. Probably related to the massive amount of data that is being streamed.

**Comment by calvinbui** [comment]
> Clicking each of the containers did load the logs quickly. But that makes sense because I use a different connection to load the logs.

This is my experience as well.  When switching between containers using the sidebar (when it does load), there is no lag either. There are no live stats here. 

I'm able to get logs going on mobile with a direct link and switching between containers using the sidebar works. However, navigating back to the home page and clicking on any element ends up freezing the entire tab again.

**Comment by amir20** [comment]
> This is my experience as well. When switching between containers using the sidebar (when it does load), there is no lag either. There are no live stats here.

This was bothering me because if stats was the real culprit then it should have been sluggish on all pages. 

Turns out it is something that was only happening on the home page. When calculating the list of containers for the search in the home UI, each stat up was triggering a ui refresh. Not great. 

There is definitely more head room for improvement here. But for now, I have fixed the bug and pushing a new build.
